src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMemberDetail.jsx

```diff
@@ -142,16 +155,71 @@ class OrganizationMemberDetail extends AsyncView {
     });
   };
 
+  removeAuthenticator = (userId, authId) => {
+    let endpoint = `/users/${userId}/authenticators/${authId}/`;
+
+    return this.api.requestPromise(endpoint, {
+      method: 'DELETE',
+    });
+  };
+
+  handle2faReset = () => {
+    let {member} = this.state;
+    let {slug} = this.getOrganization();
+
+    let user = member.user;
```

Comment:
imo: (doesn't need to be changed) - since `member` isn't being used, I would destructure `user` from `this.state.member`